WordPress/Classes/ViewRelated/SuggestionsListViewModel.swift

```diff
@@ -123,15 +123,35 @@ import CoreData
         }
         switch suggestions {
         case .users(let userSuggestions):
+            let sortedUserSuggestions = Self.sort(
+                userSuggestions: userSuggestions, by: searchQuery
+            )
             return Self.searchResultByMovingProminentSuggestionsToTop(
-                userSuggestions: userSuggestions,
+                userSuggestions: sortedUserSuggestions,
                 prominentSuggestionsIds: prominentSuggestionsIds
             )
         case .sites(let siteSuggestions):
             return .sites(siteSuggestions)
         }
     }
 
+    private static func sort(userSuggestions: [UserSuggestion], by searchQuery: String) -> [UserSuggestion] {
```

Comment:
Can you add a brief documentation of what's happening here? 🙏

Reply:
Sure. It's done. Thank you for reminding me.